assemblies/assembly-planner-configuration.adoc

```diff
@@ -258,30 +258,21 @@ Use the solver to solve a planning problem.
 .Procedure
 Provide the planning problem as an argument to the `solve()` method. The solver will return the best solution found.
 
-The following example solves the NQueens problem:
+The following example solves the timetable problem:
 
 [source,java,options="nowrap"]
 ----
-    NQueens problem = ...;
-    NQueens bestSolution = solver.solve(problem);
+    timetable problem = ...;
+    timetable bestSolution = solver.solve(problem);
 ----
 
-In this example, the `solve()` method will return an `NQueens` instance with every `Queen` assigned to a `Row`.
+In this example, the `solve()` method will return an `timetable` instance with every `timetable` assigned to a `Row`.
```

Comment:
In this example, the `solve()` method will return an `timetable` instance with every lesson assigned to a room and a timeslot. 
![image](https://user-images.githubusercontent.com/21653197/128842123-d673b723-67b0-4727-9a3e-77e495cb80c8.png)